static/app/views/performance/queues/queuesLandingPage.tsx

```diff
@@ -21,12 +26,45 @@ import {ModulePageProviders} from 'sentry/views/performance/modulePageProviders'
 import Onboarding from 'sentry/views/performance/onboarding';
 import {LatencyChart} from 'sentry/views/performance/queues/charts/latencyChart';
 import {ThroughputChart} from 'sentry/views/performance/queues/charts/throughputChart';
-import {QueuesTable} from 'sentry/views/performance/queues/queuesTable';
+import {isAValidSort, QueuesTable} from 'sentry/views/performance/queues/queuesTable';
 import {MODULE_TITLE, RELEASE_LEVEL} from 'sentry/views/performance/queues/settings';
+import {QueryParameterNames} from 'sentry/views/starfish/views/queryParameters';
+
+const DEFAULT_SORT = {
+  field: 'time_spent_percentage()' as const,
+  kind: 'desc' as const,
+};
 
 function QueuesLandingPage() {
   const organization = useOrganization();
   const onboardingProject = useOnboardingProject();
+  const location = useLocation();
+
+  const query = useLocationQuery({
+    fields: {
+      destination: decodeScalar,
+      [QueryParameterNames.DOMAINS_SORT]: decodeScalar,
+    },
+  });
+
+  const sort =
+    decodeSorts(query[QueryParameterNames.DOMAINS_SORT]).filter(isAValidSort).at(0) ??
+    DEFAULT_SORT;
```

Comment:
This is placeholder for now as I'm working on completing the table sorting. (`DOMAINS_SORT` will be replaced to something more appropriate).

Can remove from this pr if needed.